proto/marshal/rules/message.py

```diff
@@ -34,14 +34,15 @@ def to_proto(self, value):
             try:
                 # Try the fast path first.
                 return self._descriptor(**value)
-            except (TypeError, ValueError) as ex:
-                # If we have a TypeError or Valueerror,
+            except (TypeError, ValueError, AttributeError) as ex:
+                # If we have a TypeError, Valueerror or AttributeError,
                 # try the slow path in case the error
                 # was:
                 # - an int64/string issue.
                 # - a missing key issue in case a key only exists with a `_` suffix.
                 #   See related issue: https://github.com/googleapis/python-api-core/issues/227.
                 # - a missing key issue due to nested struct. See: b/321905145.
+                # - a missing key issue due to nested duration. See: b/383651746.
```

Comment:
Use external links as much as possible.

```suggestion
                # - a missing key issue due to nested struct. See: https://github.com/googleapis/proto-plus-python/issues/424.
                # - a missing key issue due to nested duration. See: https://github.com/googleapis/google-cloud-python/issues/13350.
```